### PR TITLE
doc: FederatedHPA docs cleanup

### PR DESCRIPTION
**Pull request by jwcesign**
**What type of PR is this?**
/kind cleanup
/kind documentation

**What this PR does / why we need it**:
It's unclear in FederatedHPA docs.

**Which issue(s) this PR fixes**:
Fixes #none

**Special notes for your reviewer**:
none

### CODE REVIEW

**Review by XiShanYongYe-Chang — COMMENTED** [review]
This file appears to have no matching directory:

image

File: i18n/zh/docusaurus-plugin-content-docs/current/userguide/autoscaling/federatedhpa.md [hunk 1]
```diff
@@ -37,5 +37,5 @@ FederatedHPA 是 Karmada 弹性伸缩 API 组中的一个 API。当前版本为
 如果您配置了 FederatedHPA，则可能还需要考虑运行类似于 [Cluster Autoscaler](https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler) 的集群级别自动扩缩容工具。 
 
 有关 FederatedHPA 的更多信息：
-* 阅读 [FederatedHPA 教程](../../tutorials/autoscaling-with-federatedhpa.md)以了解 FederatedHPA。
-* 阅读 [FederatedHPA 自定义指标教程](../../tutorials/autoscaling-with-custom-metrics.md)以了解 FederatedHPA。
+* 阅读 [FederatedHPA基于resource metrics(CPU/Memory)弹性伸缩](../../tutorials/autoscaling-with-resource-metrics.md) 以了解 FederatedHPA。
```
Comment (samzong) [inline i18n/zh/docusaurus-plugin-content-docs/current/userguide/autoscaling/federatedhpa.md:40]: 尽量在中文和英文之间保持空格
```suggestion
* 阅读 [FederatedHPA 基于 resource metrics(CPU/Memory) 弹性伸缩](../../tutorials/autoscaling-with-resource-metrics.md) 以了解 FederatedHPA。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/userguide/autoscaling/federatedhpa.md [hunk 2]
```diff
@@ -37,5 +37,5 @@ FederatedHPA 是 Karmada 弹性伸缩 API 组中的一个 API。当前版本为
...
+* 阅读 [FederatedHPA基于custom metrics(自定义指标)弹性伸缩](../../tutorials/autoscaling-with-custom-metrics.md) 以了解 FederatedHPA。
```
Comment (samzong) [inline i18n/zh/docusaurus-plugin-content-docs/current/userguide/autoscaling/federatedhpa.md:41]:
```suggestion
* 阅读 [FederatedHPA 基于 custom metrics (自定义指标)弹性伸缩](../../tutorials/autoscaling-with-custom-metrics.md) 以了解 FederatedHPA。
```

**Comment by jwcesign** [comment]
> This file appears to have no matching directory:

Already fixed

**Review by RainbowMango — APPROVED** [review]
/approve

Leave final decisions to @samzong @XiShanYongYe-Chang

**Pull request merged**